docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cmake \
     ../src
 RUN make VERBOSE=1 -j2
 RUN ccache -s
-RUN ctest -E "(walker:SkewNormalPDF_pe8|walker:DiagOrnsteinUhlenbeckPDF_pe8)"
+RUN ctest -E "_pe[45678]"
```

Comment:
Okay, but I still think, `Release` is sufficient on travis considering time limitations there.